### DISCUSSION TITLE
[3DS] More readable text

### DISCUSSION THREAD

**Discussion by JetAltOne** [diasurgical/DevilutionX#5247] [Ideas]
The text is absolutely unreadable
![IMG_3926](https://user-images.githubusercontent.com/110492161/184901853-e36013da-9a5c-421f-a570-9c7ac9d234d3.jpg)
e

Please consider adding GUI adjustments/Improvements

**Comment by AJenbo** [comment]
easy just replace the screen...

**Reply by JetAltOne** [reply]
> easy just replace the screen...

seems legit

**Reply by AJenbo** [reply]
I knew the screen resolution was a bit low on 3DS so originally thought it was just how it looks :sweat_smile:

**Comment by StephenCWills** [comment]
This happens if you change the resolution from the Settings menu, and should resolve itself if you restart the game or watch the intro video.

There may also be some combination of graphics settings that cause it. If it doesn't clear up for you, it would be helpful if you could share your game version and graphics settings.

**Reply by JetAltOne** [reply]
> This happens if you change the resolution from the Settings menu, and should resolve itself if you restart the game or watch the intro video.
> 
> There may also be some combination of graphics settings that cause it. If it doesn't clear up for you, it would be helpful if you could share your game version and graphics settings.

the resolution is set to default

**Reply by JetAltOne** [reply]
I have the latest version running on a 3DS XL

**Reply by AJenbo** [reply]
What is the default resolution?

**Reply by JetAltOne** [reply]
800 x 480

**Reply by JetAltOne** [reply]
tested on 800 x 432, just as bad

**Reply by qndel** [reply]
not absolutely unreadable as I can see it's a "short staff of charged bolt" 🤪

**Reply by JetAltOne** [reply]
numbers are hard to read properly

**Reply by JetAltOne** [reply]
watching the intro did fix it, but i shouldn't have to do that...

**Reply by AJenbo** [reply]
Sounds like it is failing to restore the resolution if the video is interrupted.

**Reply by JetAltOne** [reply]
seems like it

**Reply by StephenCWills** [reply]
> watching the intro did fix it, but i shouldn't have to do that...

I guess you'll have to, unless you can provide me your game version and graphics settings like I asked.

**Reply by StephenCWills** [reply]
Maybe try using these and let me know how it goes.

```ini
[Graphics]
Width=640
Height=480
Fullscreen=0
Upscale=0
Fit to Screen=0
Scaling Quality=2
Integer Scaling=0
Vertical Sync=1
Blended Transparency=1
Gamma Correction=100
Color Cycling=1
FPS Limiter=0
Show FPS=1
Show health values=0
Show mana values=0
Alternate nest art=0
Zoom=0
```

**Reply by JetAltOne** [reply]
[Graphics]
Width=800
Height=480
Fullscreen=1
Fit to Screen=1
Gamma Correction=100
Zoom=0
FPS Limiter=1
Show FPS=1
Show health values=1
Show mana values=1
Color Cycling=1
Alternate nest art=0

i have a 3DS XL

**Reply by JetAltOne** [reply]
3DS XL on Version 1.4.1
![IMG_3927](https://user-images.githubusercontent.com/110492161/184914150-bc571f60-c87e-41a1-90d6-43b087aa522e.jpg)

**Reply by StephenCWills** [reply]
Hm, it's still not happening for me. I can only reproduce the issue by changing the `Resolution`, `Fullscreen`, or `Fit to Screen` settings and then it clears up when I watch the intro video or restart the game.
